docs/getting-started/how-to/add-github-codespaces-to-your-project/index.md

```diff
@@ -61,6 +61,23 @@ git switch --create=github-codespaces
 * Repeat for the remaining files.
 * Copy each file to the `.devcontainer` folder in your project.
 
+!!! warn "Dotfiles and `.gitignore`"
+    Dotfiles are files and folders that start with a dot (`.`).
+    Many tools for browsing files and folders,
+    such as File Explorer on Windows or Finder on macOS,
+    hide dotfiles by default, so as not to clutter the display.
+
+    GitHub shows dotfiles, but removes the dots when they are downloaded.
+    Whilst this behaviour is understandable
+    ("Where did the file I just downloaded go?")
+    it is also unhelpful, as the dots are important.
```

Comment:
I feel like "Where did the file I just downloaded go?" is more associated with unhelpfulness rather than understandable behaviour.

```suggestion
    Whilst this behaviour is understandable it is also unhelpful
    ("Where did the file I just downloaded go?")
    as the dots are important.
```

Reply:
Thanks! That's not quite the meaning I had in mind, so I've just removed the question in parentheses.